Doc/reference/import.rst

```diff
@@ -333,6 +333,9 @@ modules, and one that knows how to import modules from an :term:`import path`
    Use of :meth:`~importlib.abc.MetaPathFinder.find_module` by the import system
    now raises :exc:`ImportWarning`.
 
+.. versionchanged:: 3.12
+   ``find_module()`` has been removed.  Use ``find_spec()`` instead.
```

Comment:
Should `:meth:` be used here? The Netlify preview is unavailable so I cannot check it myself.

```suggestion
   ``find_module()`` has been removed.  Use :meth:`find_spec` instead.
```

Reply:
Yep, thanks!